main.js

```diff
@@ -44,6 +44,26 @@ const handlers = {
     });
   },
 
+  handleLoadSpec: () => {
+    const options = {
+      properties: ['openFile'],
+      filters: [
+        {
+          name: 'Vega Lite Specs',
+          extensions: ['json'],
+        },
+      ],
+    };
+
+    dialog.showOpenDialog(mainWindow, options, (filenames) => {
+      if (filenames && filenames.length > 0) {
```

Comment:
Interesting that it takes multiple file names as input.  Is this because the file loader support selecting mutiple files?  If so, we should make sure it only supports selecting only one file?